jobs/gtfs-rt-parser-v2/gtfs_rt_parser.py

```diff
@@ -640,20 +643,23 @@ def parse_and_validate(
             )
         except (ScheduleDataNotFound, subprocess.CalledProcessError) as e:
             with sentry_sdk.push_scope() as scope:
-                scope.set_context("hour", json.loads(hour.json()))
+                scope.set_context("RT Hourly Aggregation", json.loads(hour.json()))
                 fingerprint: List[Any] = [
                     type(e),
                     # convert back to url manually, I don't want to mess around with the hourly class
                     base64.urlsafe_b64decode(hour.base64_url.encode()).decode(),
                 ]
                 if isinstance(e, subprocess.CalledProcessError):
                     fingerprint.append(e.returncode)
-                    # try to get the top of the stacktrace since this will be truncated; 1800 is just an estimate
                     scope.set_context(
-                        "schedule_extract", json.loads(schedule_extract.json())
+                        "First RT Extract in Hour", json.loads(hour.extracts[0].json())
+                    )
+                    scope.set_context(
+                        "Schedule Extract", json.loads(schedule_extract.json())
                     )
+                    # get the end of stderr, just enough to fit in MAX_STRING_LENGTH defined above
                     scope.set_context(
-                        "process", {"stderr": e.stderr.decode("utf-8")[-1800:]}
+                        "Process", {"stderr": e.stderr.decode("utf-8")[-2000:]}
```

Comment:
sorry but why is this `2000` instead of exactly the same as the max string length?

Reply:
No reason; I just stuck with a power of 2 for the library setting.